pyteal/ast/router.py

```diff
@@ -68,6 +68,7 @@ def __init__(self, name: Optional[str] = None) -> None:
 
     @staticmethod
     def __parse_conditions(
+        method_signature: str,
```

Comment:
nit: would be good to add a check for `method_signature == None` if `method_to_register == None`